docs/pjrt_integration.md

```diff
@@ -96,11 +117,11 @@ entry_points={
   ],
 }
 ```
-Here are examples of how openxla-pjrt-plugin is implemented using Option 2: https://github.com/openxla/openxla-pjrt-plugin/pull/119, https://github.com/openxla/openxla-pjrt-plugin/pull/120.
+Here's an example of how a PJRT plugin is implemented using Option 2: https://github.com/jax-ml/jax/tree/main/jax_plugins/cuda.
```

Comment:
This link is mentioned a couple of times in this document but I think it also fits here.